### ISSUE TITLE
Panning fires info event in WPF

### ISSUE THREAD

**Issue by npace** [pauldendulk/Mapsui#325]
After I implemented the click-to-select-country functionality discussed in #314, I noticed that whenever I drag to pan the map, the country underneath the mouse pointer gets selected, e.g. the pan action is being interpreted as a click action.
Issue #323 seems to discuss the same problem but on Android. I tried both:
 - adding Mapsui 1.3.2-beta4 in my project
 - building Mapsui from the latest commit on the master branch ([9bfbe13](https://github.com/pauldendulk/Mapsui/commit/9bfbe134907053052d656e1aad9da685f61e07a7)) 

So far, neither approach prevents the pan action from firing an info event.
I'm attaching a GIF I made by building from the master branch with the code modifications from [this comment](https://github.com/pauldendulk/Mapsui/issues/314#issuecomment-366723668):
![click-drag-bug](https://user-images.githubusercontent.com/1901561/36432238-644b4166-1662-11e8-9b72-b389546c2375.gif)

**Comment by pauldendulk** [comment]
I can confirm this is a bug. An old one. 

How did you create the animated gif? Very informative.

**Comment by npace** [comment]
After looking around the source code, I have an idea:
The MapControlMouseLeftButtonUp method in the MapControl class can be modified to check for a distance threshold before handling a click event:
```
var mousePosition = e.GetPosition(this).ToMapsui();
var distance = mousePosition.Distance(_downMousePosition.ToMapsui());
if (distance > 10)
{
    return;
}
```

The problem is that the distance value (10) is completely arbitrary and I have no idea how to choose a better one.

By the way, I made the gif using [ScreenToGif](http://www.screentogif.com) - it's an open source tool for screen capture.

**Comment by pauldendulk** [comment]
So I basically did what you suggested using the SystemParameters.MinimumHorizontalDragDistance parameter that was already used for touch info. I think in many cases click is ignored if there is any movement. Not sure if the current solution is unexpected for the user. It now show info when move is smaller than 4.

**Comment by npace** [comment]
Something seems a bit off...
Using the published stable version 1.3.2 from NuGet, the bug can still be reproduced. I tried referencing my local copy of the Mapsui repository instead, and the bug disappeared. 
Maybe something went wrong with the release of 1.3.2?

**Comment by pauldendulk** [comment]
This was not included in the 1.3.2 release, but every commit is pushed to the appveyor nuget repository. You could try that. Instructions [here](https://github.com/pauldendulk/Mapsui/wiki/NuGet-of-latest-build).
